libPhoneNumber/NBMetadataHelper.m

```diff
@@ -186,4 +193,67 @@ + (BOOL)hasValue:(NSString *)string {
   return string.length != 0;
 }
 
+#if SHORT_NUMBER_SUPPORT
+
++ (NSDictionary *)shortNumberDataMap {
+    static NSDictionary *shortNumberDataDictionary;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        // Data is a gzipped JSON file that is embedded in the binary.
+        // See GeneratePhoneNumberHeader.sh and PhoneNumberMetaData.h for details.
+        NSMutableData* gunzippedData =
+            [NSMutableData dataWithLength:kShortNumberMetaDataExpandedLength];
+
+        z_stream zStream;
+        memset(&zStream, 0, sizeof(zStream));
```

Comment:
since this essentially duplicates the other code for reading this stuff, can we move into a single function?